pom.xml

```diff
@@ -277,6 +277,37 @@
     </dependencies>
   </dependencyManagement>
 
+  <build>
+    <pluginManagement>
```

Comment:
As far as I remember adding a plugin inside `pluginManagement` does not necessarily invoke them in child modules, you must declare this plugin in child module to run.